definitions/tools/pvacseq.cwl

```diff
@@ -20,6 +20,7 @@ arguments:
 inputs:
     input_file:
```

Comment:
Not something changed in this PR, but would a more descriptive name or a `doc` line to explain what sort of file this is be useful?

Reply:
I just pushed a commit to the PR to rename this input to `input_vcf`.